docs/docset.yml

```diff
@@ -41,6 +41,7 @@ subs:
   ilm-init:   "ILM"
   dlm:   "data lifecycle management"
   dlm-init:   "DLM"
+  major-version: "9.x"
+  major-minor-version: "9.0"
```

Comment:
@karenzone might have a better input here.
I don't see versions defined in [ES docset](https://www.elastic.co/docs/deploy-manage/deploy/self-managed/install-elasticsearch-with-rpm) and [KB docset](https://www.elastic.co/docs/deploy-manage/deploy/self-managed/install-kibana-with-rpm). Is it something Logstash specific?
Do we also need `major-minor-version`?

Reply:
I think it might actually be hardcoded for ES - https://github.com/elastic/docs-content/blob/19ebcc448b9f1ad9497dd9aefd199244cd73e295/deploy-manage/deploy/self-managed/install-elasticsearch-with-rpm.md?plain=1#L60

Reply:
we don't need major-minor-version, I've checked that this attribute is only used in this docs section.